### ISSUE TITLE
Log enhancement: OCSP Stapling can't be enabled

### ISSUE THREAD

**Issue by franciscopaniskaseker** [litespeedtech/openlitespeed#177]
When you get an error like

```bash
2020-04-27 18:31:13.685936 [ERROR] [SSL] OCSP Stapling can't be enabled [X509_STORE_CTX_get1_issuer failed!].
```

will be helpful if it print which domain the problem is happening.

**Comment by litespeedtech** [comment]
That's because the SSL certificate does not include that information, the URL for OCSP verification.
This cannot be fixed in OLS software, need to fix the certificate.

**Comment by lucasRolff** [comment]
I believe that was the point, to fix the certificate, you have to know which certificate is broken in the first place. Thus as OP suggested, it would be nice to log which domain/certificate is the culprit.

**Comment by litespeedtech** [comment]
Sorry.
Yes, it should include domain/certificate information in the log.

**Comment by litespeedtech** [comment]
Fix is in 1.6.20 and 1.7.9 now.
